index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+      <div class="home-page">
+        <header>
+
+            <nav class="hamburger">
+                <div class="log"><img src="img/My Logo.png" alt="Menu"></div>
+                <ul>
+                   
+                    <li class="log"><img src="img/Union.png" alt="Menu1"></li>
+            
+                </ul>
+            </nav>
+        </header>
+        <section id="home">
+           <h4 class="presentation">I'm Mehdi jdir
+            Glad to see you!</h4>
+            <p>
+                I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hesitate to contact me.
+            </p>
+            
+                <h3>LET'S CONNECT</h3>
+                <div id="contact">
+                   <ul>
+                     <li class="icon"><a href="" target="_blank"><img src="img/twitter.png" alt="twitter link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="img/github.png" alt="github link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="img/succes.png" alt="succes link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="img/Linkedin.png" alt="Linkedin link"></a></li>
+                     <li class="icon"><a href="" target="_blank"><img src="img/Mobile.png" alt="Mobile link"></a></li>
+                   </ul>
+                </div>
+            
+        </section>
```

Comment:
- [ ] You also did great here using the section semantic tag. :+1: :+1: There are some design issues you need to fix in this section,
- [ ] The title `<h4 class="presentation">I'm Mehdi jdir Glad to see you!</h4>`  is the main title of your portfolio and it should be in an `h1` tag, I suggest you update this. :+1: :+1: 
- [ ] Looking at your project, you are missing some background design images, I'll suggest you get them from the Figma design and update your project accordingly. :+1: :100: 
- [ ] Also your design seems to have more spacing on both sides, I'll suggest you check the correct spacing or margin value from the Figma file and update your CSS accordingly. :smile: :wave: 

- - FIGMA DESIGN POINTING OUT THE BACKGROUND IMAGES:
![Selection_056](https://github.com/medijdir/portfolio_project/assets/107632780/d159e468-e962-405f-9e49-284774d8ff6a)

- - YOUR CURRENT DESIGN WITH THE SPACES:
![Selection_057](https://github.com/medijdir/portfolio_project/assets/107632780/74b19d87-2fc7-4ddc-8b4e-4f36f78bb7bc)